Sources/BraveWallet/WalletStrings.swift

```diff
@@ -4752,5 +4752,33 @@ extension Strings {
       value: "Details",
       comment: "The title of the details view for Sign In With Ethereum/Brave Wallet requests."
     )
+    public static let transactionSummaryIntentLabel = NSLocalizedString(
+      "wallet.transactionSummarySendType",
```

Comment:
wrong key

Reply:
Good catch! Epdated in [2f5df77](https://github.com/brave/brave-ios/pull/8390/commits/2f5df77d01dae694733aeb704398f1fb5b7cfbae)